README.org

```diff
@@ -255,22 +221,45 @@ The =Provider.ItemType.MOVIE= has the following additional fields:
 #+BEGIN_SRC
 var detail = {
 //--- including all the fields of the generic detail object
-    torrents: [Object],        // torrents object
-    trailer: String,           // url of the trailer, formated for butter-streamers
+    torrents: {                       // torrents object
+        [720p,1080p]: {               // the quality of the episode
+            filesize: String,         // human readable size
+            size: Number,             // size in bytes
+            peers: Number,             // the amount of peers
+            seeds: Number,             // the amount of seeds
+            url: String               // the url to the movie to download
```

Comment:
this would be a url compatible with what @vankasteelj and i are working for butter-streamers, basically torrent/magnet/http with a syntax compatible with what they implemented in webtorrent.